public/app/plugins/panel/canvas/editor/options.ts

```diff
@@ -120,4 +121,22 @@ export const optionBuilder: OptionSuppliers = {
       },
     });
   },
+
+  addDirection: (builder, context) => {
+    const category = ['Direction'];
```

Comment:
May make sense to call this "Connection Direction" or something like that. It's a bit ambiguous at the moment.

Reply:
I think in this case, because it's in the context of a selected connection, I'll leave it as is for now to be consistent with its siblings.
![image](https://github.com/grafana/grafana/assets/60050885/8ebb7090-d6cb-474e-a049-0bf78054271e)

Reply:
Or perhaps "Arrow Direction" would suffice?